### ISSUE TITLE
modules directores must be a string (instead of array)

### ISSUE THREAD

**Issue by lironess** [nkt/atom-autocomplete-modules#50]
on my webpack (v1.12.12) config I entered some module directories (array of, like written in documentation):

```
  resolve: {
    root: [appPath, testPath],
    ...
    modulesDirectories: [modulesPath, appPath]
  },
```

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [autocomplete-modules](https://github.com/nkt/atom-autocomplete-modules) package, v1.6.3
### Stack Trace

Uncaught TypeError: Path must be a string. Received [ '/Users/lironshmuel/Workspace/hwmarket/app',
  '/Users/lironshmuel/Workspace/hwmarket/test' ]

```
At path.js:7

TypeError: Path must be a string. Received [ '/Users/lironshmuel/Workspace/hwmarket/app',
  '/Users/lironshmuel/Workspace/hwmarket/test' ]
    at assertPath (path.js:7:11)
    at Object.join (path.js:1253:7)
    at /Users/lironshmuel/.atom/packages/autocomplete-modules/src/completion-provider.js:169:53
    at Array.map (native)
    at CompletionProvider.lookupWebpack (/Users/lironshmuel/.atom/packages/autocomplete-modules/src/completion-provider.js:168:42)
    at CompletionProvider.getSuggestions (/Users/lironshmuel/.atom/packages/autocomplete-modules/src/completion-provider.js:60:26)
    at /Users/lironshmuel/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:197:45
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (/Users/lironshmuel/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:177:15)
    at AutocompleteManager.getSuggestionsFromProviders (/Users/lironshmuel/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (/Users/lironshmuel/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:171:6)
    at /Users/lironshmuel/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1
```
### Commands

```
     -0:36.4.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:33.6.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:33.6.0 core:cancel (atom-text-editor.editor.is-focused)
  4x -0:33.5.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:32.3.0 core:cut (atom-text-editor.editor.is-focused)
     -0:32 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:30.8.0 core:move-up (atom-text-editor.editor.is-focused)
  2x -0:30.2.0 core:move-down (atom-text-editor.editor.is-focused)
  3x -0:29.6.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:28.1.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:27.9.0 core:paste (atom-text-editor.editor.is-focused)
  2x -0:27.3.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:26.5.0 editor:select-to-end-of-line (atom-text-editor.editor.is-focused)
  2x -0:23 core:move-down (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:22.7.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:05.9.0 core:backspace (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "git-difftool",
      "nuclide"
    ],
    "themes": [
      "nucleus-dark-ui",
      "atom-dark-fusion-syntax"
    ]
  },
  "autocomplete-modules": {
    "webpack": true
  }
}
```
### Installed Packages

``` coffee
# User
advanced-open-file, v0.16.4 (inactive)
atom-beautify, v0.29.12 (active)
atom-dark-fusion-syntax, v1.2.4 (active)
atom-ternjs, v0.15.0 (active)
autocomplete-modules, v1.6.3 (active)
autocomplete-plus, v2.31.3 (active)
hyperclick, v0.0.38 (active)
js-hyperclick, v1.5.0 (active)
linter, v1.11.16 (active)
linter-eslint, v7.3.2 (active)
linter-jsonlint, v1.3.0 (active)
merge-conflicts, v1.4.4 (active)
nucleus-dark-ui, v0.9.5 (active)
nuclide, v0.168.0 (inactive)
pigments, v0.34.0 (active)
project-plus, v0.9.0 (active)
react, v0.16.0 (active)
split-diff, v1.0.4 (inactive)
Sublime-Style-Column-Selection, v1.7.2 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (inactive)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.3 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by samet-karaibryamov** [comment]
I believe this was resolved by #60.

**Comment by jonyeezs** [comment]
@nkt can be closed as mentioned above.
